text/chapter6.md

```diff
@@ -346,7 +346,7 @@ data Shape
   | Text Point String
 ```
 
-5. (Medium) Derive (via `Generic`) a `Show` instance for `Shape`. How does the amount of code written and `String` output compare to `showShape` from the previous chapter? _Note_: You may instead write this instance manually, but you'll need to pay close attention to the output format expected by the tests.
+5. (Medium) Derive (via `Generic`) a `Show` instance for `Shape`. How does the amount of code written and `String` output compare to `showShape` from the previous chapter? _Note_: You may instead write this instance manually, but you'll need to pay close attention to the output format expected by the tests. _Hint_: See the [Deriving from `Generic`](https://github.com/purescript/documentation/blob/master/guides/Type-Class-Deriving.md#deriving-from-generic) section of the [Type Class Deriving](https://github.com/purescript/documentation/blob/master/guides/Type-Class-Deriving.md) guide.
```

Comment:
Why include the _note_ to optionally write it manually when the exercise clearly instructs to derive `Generic`.

Reply:
The note was to emphasize that there are two possible strategies for creating these instances, but I guess that fact should be clear enough anyway for folks reading the guide. Deleted the note.